### ISSUE TITLE
Astro support

### ISSUE THREAD

**Issue by Exerra** [berrysauce/ingredients#16]
Hello!

You can check if a website is built using [Astro](https://astro.build/) by checking the stylesheet links for the `/_astro` path. Also, you can check if [Astro image optimisation](https://docs.astro.build/en/guides/integrations-guide/image/) is enabled by checking if an image is hosted under the `/_astro` path (every other image or static file would just be under `/` so it doesn't change, but image optimisation creates a new copy with a seperate id and puts it under the `/_astro` folder).

![Astro stylesheet links](https://share.exerra.xyz/1ay9ln8.png)
![Astro optimised image](https://share.exerra.xyz/R6JEIrf.png)

I did not find any Astro headers, but you can also use the generator meta property for yet another check (though it comes with the boilerplate so it isn't auto generated at build time, so a bunch of websites might have it removed)

![Astro generator tag](https://share.exerra.xyz/UdEnSvZ.png)

Edit: So these screenshots were grabbed from [my website](https://exerra.xyz) which has Astro v2, but if I go to my blog (with Astro v1), it doesn't have the `/_astro` directory.

HOWEVER, it also doesn't use Tailwind (unlike my portfolio website) so Astro compiles classes for each of the elements affected by CSS. Classes look like `astro-[ID]`. I checked out [astro.build](https://astro.build) out and they do not use Tailwind, so they have the astro build time classes, but their stylesheets use the `/_astro` directory, just like my portfolio website. Oh, and their website is on v2.

![](https://share.exerra.xyz/MOEzmLc.png)

So tldr, Astro v1 uses `/assets/` for assets generated during build time, Astro v2 uses `/_astro/`. Also, if a website is not using Tailwind classes, but instead uses just CSS, Astro generates an `astro-[ID]` class for each styled element.

**Comment by berrysauce** [comment]
Thanks for the help! Astro is now included! 😊
